main_app/templates/main_app/helpline_numbers.html

```diff
@@ -208,9 +212,1025 @@ <h1 class="wb-stl-heading1">
                                     aria-hidden="true"></i></a></span></strong></p>
                     </td>
                 </tr>
-
             </tbody>
         </table>
+        
+
+
+        
+            
+            
+
+    </div>
+
+
+    
+        
+    <div class="column" style="margin-top: 50px; text-shadow: 2px 2px grey; text-align: center; margin-bottom: 30px;">
+        <h1 >
+            <strong>
+                <font color="black">WOMEN HELPLINE NUMBERS STATEWISE</font>
+            </strong>
+            <img src="{% static 'Images/helpline.png' %}" alt="24 * 7" height="50vw">
+        </h1>
+    </div>
+    
+
+    <div class="container">
+            
+        <ul class="collapsible popout" data-collapsible="accordion">
+            <li>
+                <div class="collapsible-header">
+                    <i class="material-icons">arrow_drop_down</i>
+                    <p style="height: 30px; padding-top: 35px;" width="275" class="wb-stl-normal"><span class="wb-stl-special"><span> &nbsp; &nbsp; <h3><strong>DELHI</strong></h3></span></span></p>
```

Comment:
- [ ] Change the font style of all the headings. Make it similar to the font style of `Women Helpline - All India`.
- [ ] Add hovering effect to the block of heading.